test/testautomation_events.c

```diff
@@ -48,11 +48,18 @@ static int events_pushPumpAndPollUserevent(void *arg)
     SDL_Event event1;
     SDL_Event event2;
     int result;
+    SDL_WindowID ref_windowid = (SDL_WindowID) SDLTest_RandomUint32();
+    Sint32 ref_code = SDLTest_RandomSint32();
+
+    /* Flush all events */
+    while (SDL_PollEvent(&event2)) {
```

Comment:
```suggestion
    SDL_FlushEvents(SDL_EVENT_FIRST, SDL_EVENT_LAST);
```

Reply:
Seems like a good use in the test suite.
The function was not used in *any* test.